funcx_sdk/funcx/tests/test_basic.py

```diff
@@ -49,6 +52,11 @@ def test_non_blocking(fxc, endpoint):
             break
 
 
+def test_malformed_service_address(fxc, endpoint):
+    with pytest.raises(InvalidServiceAddress):
+        FuncXClient(funcx_service_address="http:/k8s-dev.funcx.org/api/v1")
+
+
```

Comment:
Could you add a test here that checks whether the ws_uri is correctly formulated by the FuncXClient with no args supplied?